09_maps/main.go

```diff
@@ -223,6 +226,31 @@ func calcBMI(mapInput map[string]int) {
 // TODO: Create a function that takes an array of maps as an argument and
 // prints the average BMI of the people in the array.
 
+// create function
+// accept array of maps as input
+// loop through all arrays (e.g. "people" array of maps)
+// calculate BMI (calcBMI function)
+// add the sum of all array elements
+// increase counter
+// divide through the amount of array elements
+// print result = avg BMI
+func calcAvgBMI(arrayOfMapInput []map[string]int) {
+	var counter int
+	var bmiSum float64
+	var avgBMI float64
+	var lenConvert int
+	var lenConverted float64 = float64(lenConvert)
+	for {
+		if counter > len(arrayOfMapInput) {
+			break
+		}
+		bmiSum += calcBMI(arrayOfMapInput[counter])
+		counter++
+	}
+	avgBMI = bmiSum / lenConverted
+	fmt.Println(avgBMI)
+}
```

Comment:
@nobe4 Unsure how to correctly run the `calcBMI` function inside the `calcAvgBMI` one.
Currently I get `calcBMI(arrayOfMapInput[counter]) (no value) used as valuecompiler[TooManyValues])` as error on line 247.
It seems I'm putting too many values into the function call.

Do I maybe have to remove the loop out of the function and separate the function call?